src/main/drivers/at32/dshot_bitbang_stdperiph.c

```diff
@@ -0,0 +1,292 @@
+/*
```

Comment:
Please use the standard ./default_licence.md content for new files.

Reply:
shall i update this in the generic stdperiph.c file as well?